user_tools/src/spark_rapids_tools/tools/qualx/qualx_main.py

```diff
@@ -153,6 +153,45 @@ def _get_model(platform: str,
     return xgb_model
 
 
+def _get_calib_params(platform: str,
+                      model: Optional[str] = None,
+                      variant: Optional[str] = None) -> Dict[str, float]:
+    """
+    Load calibration params for the model, to be used similar to `_get_model()`.
+    Corresponding to model path `/path/to/{model_name}.json`, load calib params
+    from `/path/to/{model_name}_calib.cfg`
+
+    The "model" argument has a precedence over the other input options. If it is defined, this function checks
+    if it is a valid path to an XGBoost model or a string literal matching a pre-trained model name.
+    If "model" is not defined, then the pre-trained model matching "platform" will be used.
+
+    Parameters
+    ----------
+    platform: str
+        Name of the platform used to define the path of the pre-trained model.
+        The platform should match a JSON file in the "resources" directory.
+    model: str
+        Either a path to a model file or the name of a pre-trained model.
+        If the input is a string literal that cannot be a file path, then it is assumed that the file
+        is located under the resources directory.
+    variant: str
+        Value of the `sparkRuntime` column from the `application_information.csv` file from the profiler tool.
+        If set, will be used to load a pre-trained model matching the platform and variant.
+
+    Returns
+    -------
+    Calibration parameters dict
+    """
+    model_path = _get_model_path(platform, model, variant)
+    calib_path = model_path.parent / Path(model_path.stem + '_calib.cfg')
+    logger.debug('Loading calib params from: %s', calib_path)
+    calib_params = {'c1': 1.0, 'c2': 1.0, 'c3': 0.0}
+    if calib_path.exists():
+        with open(calib_path, 'r', encoding='utf-8') as f:
+            calib_params = json.load(f)
```

Comment:
I'd move L187 logging to L190, and add another log statement if the `calib_path` doesn't exist, e.g. "No calib params found" or "Using default calibration".

Reply:
Actually, the log statement for "not exist" may not be necessary (or should only be a debug log), since I see that this is used in the "normal" code path multiple times.